nuxeo-drive-client/nxdrive/wui/conflicts.py

```diff
@@ -32,6 +117,7 @@ def get_errors(self):
 
     @QtCore.pyqtSlot(result=str)
     def get_conflicts(self):
+        log.warn("Retrieving conflicts ...")
```

Comment:
this code would also be simplified fi retrieving all users once per day.
It will simply get the full name from the db. 
A user's full name will be retrieved when a doc details are retrieved with GetFileSystemItem automation API, which I think is being called every time there is a sync operation which could possibly generate a conflict. This way, the UI does need to these time delays to retrieve anything...